modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/workflow.jsp

```diff
@@ -257,21 +257,16 @@ if (tabs1.equals("published")) {
 		['aui-base']
 	);
 
-	Liferay.provide(
-		window,
-		'<portlet:namespace />editWorkflow',
-		function (uri) {
-			var A = AUI();
-
-			var WIN = A.config.win;
-
-			Liferay.Util.openWindow({
-				id: A.guid(),
-				refreshWindow: WIN,
-				title: '<liferay-ui:message key="workflow" />',
-				uri: uri,
-			});
-		},
-		['liferay-util']
-	);
+	window['<portlet:namespace />editWorkflow'] = function (uri) {
```

Comment:
Same as we went through this last time.

`editWorkflow` does indeed depend on `liferay-util` and there's no guarantee it will be loaded before the `editWorkflow` method is called from the inline `onClick` method in the button in this same jsp.

2 simple options:
- Add the button click listener ONLY when `liferay-util` is guaranteed to be loaded (another script)
- Make sure you load `liferay-util` maybe using `AUI().use()` to wrap the code that depends on it.

Reply:
A variant of second options would be to wrap it all in `<aui:script use="liferay-util">`. 

Although, I don't see that we are doing this is a lot of `Liferay.Util.openWindow` usages. I am not sure if there is some mechanism that prevents race condition, or we are just hoping that user does not click before the script loads.